src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
```

Comment:
```suggestion
  let result = days * 40;
```

You can perform this repeated calculation only once before checking discount.